src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $cell-width * 7 + $gap * 6;
+  padding: 30px $set-padding 0;
+  gap: $gap;
+
+  @each $day, $number in $days {
+    &--start-day--#{$day} :nth-child(-n + 1) {
+      margin-left: calc(
+        (#{$gap } * (#{$number} - 1)) + (#{$cell-width} * (#{$number} - 1))
+      );
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__cell {
+    width: $cell-width;
+    height: 100px;
+    box-sizing: border-box;
+    font-size: 30px;
+    line-height: 98px;
+    text-align: center;
+    cursor: pointer;
+    border: 1px solid $color-black;
+    background-color: $border-color;
+    transition: all 0.5s ease-in;
```

Comment:
It is recommended to specify the exact properties instead of `all` 
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/8620b80c-56c3-4c28-9f4c-50ceef683ce3)
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

Reply:
Done. Is changed to `transform ` instead `all`.